### PR TITLE
FW-548. Scheduling function depending on usage of cells

### PR DESCRIPTION
**Pull request by changtengfei**

### CODE REVIEW

File: openstack/02a-MAClow/IEEE802154E.c [hunk 1]
```diff
@@ -927,7 +931,7 @@ port_INLINE void activity_ti1ORri1() {
          // assuming that there is nothing to send
          ieee154e_vars.dataToSend = NULL;
          // check whether we can send
-         if (schedule_getOkToSend()) {
+         if (schedule_getOkToSend() || cellType==CELLTYPE_TX) {
```
Comment (twatteyne) [inline openstack/02a-MAClow/IEEE802154E.c]: you are bypassing the `schedule_getOkToSend()` function by putting the `||` statement!

File: openstack/02b-MAChigh/schedule.c [hunk 2]
```diff
@@ -461,6 +464,45 @@ bool schedule_isSlotOffsetAvailable(uint16_t slotOffset){
    return TRUE;
 }
 
+uint8_t schedule_getUsageStatus(scheduleEntry_t* entry){
```
Comment (twatteyne) [inline openstack/02b-MAChigh/schedule.c]: add a comment. I don't understadn what a "usage status" is. The name is not clear.

File: openstack/02b-MAChigh/schedule.c [hunk 3]
```diff
@@ -461,6 +464,45 @@ bool schedule_isSlotOffsetAvailable(uint16_t slotOffset){
    return TRUE;
 }
 
+uint8_t schedule_getUsageStatus(scheduleEntry_t* entry){
+    uint8_t  count;
+    uint16_t bm;
```
Comment (twatteyne) [inline openstack/02b-MAChigh/schedule.c]: `bm` -> `bitmap`

**Comment by changtengfei** [comment]
Sorry. This pull request is not supported to be merged in develop branch. I will close this one and do a pull request on develop_FW-548 branch

**Pull request closed without merging**